npm/webpack-preprocessor/package.json

```diff
@@ -26,6 +27,8 @@
     "loader-utils": "^2.0.0",
     "lodash": "^4.17.20",
     "md5": "2.3.0",
+    "merge-source-map": "^1.1.0",
```

Comment:
Why did you decide to use `merge-source-map` vs a more recently updated dependency? Looks like this isn't being maintained.

Also, are we still only support dependency in cy.origin when using webpack? I thought recent changes removed the need to use the custom API / webpack logic?

Reply:
> Why did you decide to use merge-source-map vs a more recently updated dependency? Looks like this isn't being maintained.

Do you have any suggestions? This is the only I could find that works.

> Also, are we still only support dependency in cy.origin when using webpack? I thought recent changes removed the need to use the custom API / webpack logic?

Yes, it still requires the webpack preprocessor. The recent changes only removed the need for `Cypress.require()`.

Reply:
Since we're patching this, why don't we lock it down and don't pull it in optimistically.

Reply:
> Since we're patching this, why don't we lock it down and don't pull it in optimistically.

Addressed in https://github.com/cypress-io/cypress/pull/24367/commits/844950ab68737202abbb5491cc248bd705ae06f1